OpenBullet/Pages/Dialogs/DialogSelectWordlist.xaml.cs

```diff
@@ -63,5 +67,30 @@ private void ListViewItem_KeyDown(object sender, System.Windows.Input.KeyEventAr
             if (e.Key == System.Windows.Input.Key.Enter)
                 selectButton_Click(this, null);
         }
+
+        private void importWordlistButton_MouseDown(object sender, MouseButtonEventArgs e)
+        {
+            OpenFileDialog ofd = new OpenFileDialog();
+            ofd.Filter = "Wordlist file | *.txt";
+            ofd.FilterIndex = 1;
+            ofd.ShowDialog();
+            try
+            {
+                // Build the wordlist object
+                var wordlist = new Wordlist(Path.GetFileNameWithoutExtension(ofd.FileName), ofd.FileName, Globals.environment.WordlistTypes.First().Name, "");
+
+                // Get the first line
+                var first = File.ReadLines(wordlist.Path).First();
+
+                // Set the correct wordlist type
+                wordlist.Type = Globals.environment.RecognizeWordlistType(first);
+
+                // Add the wordlist to the manager
```

Comment:
Maybe fix this comment to // Add the wordlist to the Runner
otherwise it's misleading. Thanks.

Reply:
btw is it just me or to fix a commit I gotta close it and resubmit a new one ? xD

Reply:
I think you can just fix it without making a new one! Look a bit online there's surely a guide somewhere